src/logger.rs

```diff
@@ -22,11 +22,17 @@ pub async fn logger_main(
     // If debug file is configured, prepare an asynchronous buffered file writer.
     let mut debug_file = None;
     if let Some(file_path) = debug_file_path {
+        // Allocate a smaller buffer (64K) when not logging response bodies.
+        let buffer_capacity = if configuration.no_debug_body {
+            64 * 1024
```

Comment:
The default size is 8K, but it I don't want to risk truncating debug logs artificially -- for example, if someone is dumping a large POST array, or there's lots of response headers.